replication.go

```diff
@@ -292,15 +244,15 @@ func (cm *ContentManager) tryAddContent(cb *contentStagingZone, c util.Content)
 
 	// if this bucket is being consolidated, do not add anymore content
 	if cb.IsConsolidating {
-		return false, nil
+		return false, errors.New("cannot add content while bucket is consolidating")
 	}
 
 	if cb.CurSize+c.Size > cb.MaxSize {
-		return false, nil
+		return false, errors.Errorf("cannot add content because resulting size would exceed max bucket size of %d bytes", cb.MaxSize)
 	}
 
 	if len(cb.Contents) >= cb.MaxItems {
-		return false, nil
+		return false, errors.Errorf("cannot add content because resulting number of items would exceed max bucket limit of %d items", cb.MaxItems)
```

Comment:
no, don't return errors in these places - we don't want to completely fail here, we want a new staging zone to be created if content cannot be added to the current zone